users/tests/test_unit_users_apiview.py

```diff
@@ -12,26 +12,29 @@
 class UserListTests(TestCase):
     def setUp(self):
         self.user = CustomUser(
-            email="testuser@codepoets.it", password="newuserpasswd", first_name="John", last_name="Doe", country="PL"
+            email="testuser@codepoets.it",
```

Comment:
I see it was there before, but I think you can now use safely values generated by factory here, so you can remove `email`, `first_name` and `last_name`

Reply:
If this tests Django views please move this tests to `test_views.py`, we will then clearly see how much is left to change :)